src/FixedDataTableNew.react.js

```diff
@@ -312,6 +324,10 @@ var FixedDataTable = createReactClass({
       this._shouldHandleWheelX,
       this._shouldHandleWheelY
     );
+    this._didScroll = throttle(
+      this._didScroll,
+      this.props.onScrollThrottle || 0
```

Comment:
@dennisberko So I'm looking at the docs for throttle (https://lodash.com/docs/2.4.2#throttle), and these params strike me as confusing. I expected the first param to be a handler function, and the second param to be a wait time in ms. 

Thoughts on renaming these params for clarity?

Reply:
As for the _didScroll, I tried to be consistent with the other functions `_didScrollStart` and `_didScrollEnd`. What's happening here is that we are wrapping the function with a throttle.

I could change onScrollThrottle to onScrollThrottleMs however I did add documentation to the param
```
    /**
     * Number of milliseconds to throttle the onScroll event
     */
    onScrollThrottle: PropTypes.number,
```

As for the `|| 0` I generally try to avoid passing null/undefined around, it has a tendency of causing bugs but seems lodash supports taking undefined as the second param so I could get rid of it

Reply:
@dennisberko I do think onScrollThrottleMs > onScrollThrottle, as the latter sounds like a handler function. But yes, I agree that documenting it helps, so I'll leave it up to your judgment.

For the `|| 0` part, I meant putting a default other than 0. e.g. `|| 100`. Thoughts?